### PR TITLE
Fixed error on parsing PID for status command.

### PR DESCRIPTION
**Pull request by govint**
Also added a unit test for status command.

### CODE REVIEW

**Comment by govint** [comment]
Fixes #633

File: esx_service/cli/vmdkops_admin.py [hunk 1]
```diff
@@ -591,7 +591,8 @@ def get_service_status():
         if output[2] == "not":
             return NOT_RUNNING_STATUS
 
-        pid = output[3].split("=")[1]
```
Comment (msterin) [inline esx_service/cli/vmdkops_admin.py]: please add a comment with expected result example (makes easy to understand and check the code).

Comment (govint) [inline esx_service/cli/vmdkops_admin.py]: This was the error:

File "/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py", line 594, in get_service_status
pid = output[3].split("=")[1]
TypeError: a bytes-like object is required, not 'str'

**Comment by govint** [comment]
Sure, and this is the fixed output:

python /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py status
Version: 0.7.3f2c66a-0.0.1
Status: Running
Pid: 5585696
Port: 1019
LogConfigFile: /etc/vmware/vmdkops/log_config.json
LogFile: /var/log/vmware/vmdk_ops.log

**Comment by msterin** [comment]
the fix looks good and good to go when CI passes

**Comment by pdhamdhere** [comment]
You may want to check CI, vmdkopsAdmin testStatus test is failing.

**Comment by govint** [comment]
Its a different test thats failing (vmdkops_admin_sanity_test.py), the new
test was added to vmdkops_admin_test.py

**Comment by govint** [comment]
More byte to string handling in test code. Fixed.

**Comment by govint** [comment]
Restarted CI test for this change I'll check and update.

**Pull request merged**